### DISCUSSION TITLE
What is `path-to-ligature-repo`?

### DISCUSSION THREAD

**Discussion by refaelsh** [mickeynp/ligature.el#55] [Q&A]
Sorry if it is a strange question, but what is `path-to-ligature-repo`? I see it in all the Wiki examples. Like this:
```elisp
:load-path "path-to-ligature-repo"
```
What should I put instead of `path-to-ligature-repo`?

Thanks in advance :-)

**Comment by izuzu-izuzu** [comment]
If you installed this package through MELPA, then you shouldn't need to mention `:load-path` at all:

```elisp
(use-package ligature
  :config
  ;; Enable the "www" ligature in every possible major mode
  (ligature-set-ligatures 't '("www"))
  ;; Enable all Cascadia Code ligatures in programming modes
  (ligature-set-ligatures 'prog-mode '("|||>" "<|||" "<==>" "<!--" "####" "~~>" "***" "||=" "||>"))
  ;; Enables ligature checks globally in all buffers.
  (global-ligature-mode t))
```

If you installed this package by cloning the repository, however, then replace `"path-to-ligature-repo"` with the path string. For example, if you cloned this repository into your `/home/refaelsh` folder, then replace `"path-to-ligature-repo"` with `"/home/refaelsh/ligature.el"`:

```elisp
(use-package ligature
  :load-path "/home/refaelsh/ligature.el"
  :config
  ;; Configure to your heart's content
  ;; ...
  )
```

I'm not entirely sure how path strings are handled in Windows, so if you're using Windows you can just install through MELPA and skip the hassle! 😅
